runtime/amplitude/src/xcm_config.rs

```diff
@@ -28,7 +28,7 @@ use xcm_executor::{
 	XcmExecutor,
 };
 
-use runtime_common::{parachains::kusama::asset_hub, RelativeValue};
+use runtime_common::{parachains::asset_hub, RelativeValue};
```

Comment:
is it supposed to be `use crate::definitions::asset_hub;` ?

Reply:
Yes! Thanks. We later moved the `asset_hub` definition to each chain.